kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -537,8 +524,9 @@ public Subscription create(Subscription subscription) {
     }
 
     public List<Subscription> findPendingSubscriptionsFromDate(DateTime startDate, int page, int pageSize) {
-        return subscriptionDataService.findByStatusAndStartDate(SubscriptionStatus.PENDING_ACTIVATION, startDate,
-                new QueryParams(page, pageSize));
+        return subscriptionDataService.findByStatusAndStartDate(SubscriptionStatus.PENDING_ACTIVATION, startDate, new QueryParams(page, pageSize));
     }
 
+    @CacheEvict(value = {"pack-type", "pack-name", "pack-all" }, allEntries = true)
```

Comment:
We still will need to expire the cache.  This will evict the entries on the server were the new pack was created, but the other servers will still have stale cached entries.